install_files/ansible-base/securedrop-configure.yml

```diff
@@ -110,6 +110,12 @@
       default: ""
       private: yes
 
+    - name: securedrop_supported_locales
+      # the list is from the securedrop/translations repository
+      prompt: Space separated list of supported locales (ar, de_DE, en_US, fr_FR, nb_NO, nl)
```

Comment:
Hmm, this is slightly confusing since this says space-separated list but then there's a comma-separated list. Maybe this could be `(ar de_DE en_US fr_FR nb_NO nl)` for clarity (or we can use `split(',')` below on line 185). 

Also, what this does currently is specify additional locales, so we could say this explicitly for clarity, e.g "Space separated list of additional locales to support (`ar de_DE fr_FR nb_NO nl`)"

Reply:
Removed the **,** good idea ! Also changed the wording but kept **en_US** in the list. I think there is value in listing all available locales even though **en_US** will always be there.